kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -1448,6 +1455,22 @@ showDotGraph gr =
         . Graph.graphToDot (graphParams gr)
         $ gr
 
+-- | A version of @showDotGraph@ that catches any exceptions.
+showDotGraphCatchException ::
+    From axiom AttrLabel.Label =>
+    From axiom RuleIndex =>
+    Gr CommonClaimState (Maybe (Seq axiom)) ->
+    IO ()
+showDotGraphCatchException gr =
+    catch (showDotGraph gr) $ \(e :: SomeException) ->
```

Comment:
Let's not catch any exception here, we should just catch the exception thrown in this case.

Reply:
Ok, I fixed this so that we are now only catching exceptions coming from from the `GVProgramExc` constructor.  If we wanted to narrow this down more we would need to inspect the message string, which seems a bit brittle to me, but let me know what you think.

Reply:
Looks good, thanks!